v2/serviceaccount/token.go

```diff
@@ -0,0 +1 @@
+package serviceaccount
```

Comment:
Q: how is this used?

Reply:
this will be used for bearer token generation functions, currently working on this

Reply:
add a comment in file.

Reply:
Added comment